components/FeeDetails/DetailedEstimates.tsx

```diff
@@ -11,17 +12,45 @@ const DetailedEstimates: FC = () => {
 
     const parsedFee = fee && parseFloat(Number(fee.walletFee).toFixed(fromCurrency?.precision))
     const currencyName = fromCurrency?.display_asset || fromCurrency?.asset || " "
+    const a = fee.avgCompletionTime?.split(':');
+    const seconds = a && (+a[0]) * 60 * 60 + (+a[1]) * 60 + (+a[2]);
```

Comment:
What's this? a?? 
If calculating seconds or something like that move to a utility file as a method with appropriate name